e2e-tests/src/test/rewards.rs

```diff
@@ -40,23 +41,151 @@ fn get_non_reserved_members_for_session(config: &Config, session: SessionIndex)
     non_reserved.iter().map(account_from_keypair).collect()
 }
 
-pub fn disable_node(config: &Config) -> anyhow::Result<()> {
-    const MAX_DIFFERENCE: f64 = 0.05;
-    const VALIDATORS_PER_SESSION: u32 = 4;
+fn get_member_accounts(config: &Config) -> (Vec<AccountId>, Vec<AccountId>) {
+    (
+        get_reserved_members(config)
+            .iter()
+            .map(account_from_keypair)
+            .collect(),
+        get_non_reserved_members(config)
+            .iter()
+            .map(account_from_keypair)
+            .collect(),
+    )
+}
 
+fn validators_bond_extra_stakes(config: &Config, additional_stakes: Vec<Balance>) {
+    let node = &config.node;
     let root_connection = config.create_root_connection();
 
-    let sessions_per_era = get_sessions_per_era(&root_connection);
-
-    let reserved_members: Vec<_> = get_reserved_members(config)
+    let validator_seeds = get_validators_seeds(config);
+    let controller_seeds: Vec<_> = validator_seeds
         .iter()
-        .map(account_from_keypair)
+        .map(|v| format!("{}//Controller", v))
         .collect();
-    let non_reserved_members: Vec<_> = get_non_reserved_members(config)
+
+    let controller_accounts_key_pairs = accounts_seeds_to_keys(&controller_seeds);
```

Comment:
Maybe we can keep this `knowledge` of which key is what in `config.rs/accounts.rs` using that `NodeKeys` struct or something similar, i.e. add method `get_node_keys -> Vec<NodeKeys>`. No more questions about keys...:)